### ISSUE TITLE
Append HTML content

### ISSUE THREAD

**Issue by cinthyargc** [dita-ot/dita-ot#3337]
### Description
I would like to append/embed the content of one HTML file to my PDF or HTML. Is this posible through DITA-OT?

I am using this link as a reference: https://www.oxygenxml.com/dita/styleguide/webhelp-feedback/Artefact/Maps/c_Non-DITA_Topic_References.html

But I am unable to make it work.

I have tried with below ditamap, but even when scope=local, the content of the HTML is not appended.

```xml
...
    <appendix navtitle="Report">
      <topicref format="html" href="content.html" locktitle="yes" navtitle="Report" scope="peer"/>
    </appendix>
    <backmatter>
....
```

**Environment:**
MacOs
DITA-OT 3.3

Thank you!

**Comment by raducoravu** [comment]
The DITA OT has a plugin called "org.lwdita" which tries to interpret references to HTML documents (with local scope) as DITA content.
Are you using the DITA OT downloaded from the DITA OT web site or the DITA OT bundled with Oxygen XML Editor. If the latter, what version of Oxygen are you using?

**Comment by cinthyargc** [comment]
Thank you so much for your response @raducoravu!

It seems like` org.lwdita` plug-in is being called, but it is always returning an error:

```
 [gen-list] Type error at char 4 in xsl:apply-templates/@select on line 17 column 43 of html2dita.xsl:
 [gen-list]   XTTE0570: An empty sequence is not allowed as the value of variable $dita
 [gen-list] [DOTJ013E][ERROR] Failed to parse the referenced file 'file:/.../ditamapfile/report.html'.: An empty sequence is not allowed as the value of variable $dita
```

And the content of my html is not being appended. I am using DITA-OT downloaded from the DITA-OT site. Tested with 3.3 and 3.3.2, both are giving the same error. Read a little bit about #2947, not sure if it's related.

**Comment by cinthyargc** [comment]
@raducoravu , I have managed to bypass that error, it looks like my html had some special tags, I went with a simpler html:

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Report</title>
    <meta charset="UTF-8"/>
  </head>
  <body>
    <h1>This is the report</h1>
  </body>
</html>
```

But, it is now failing on `topic-fragment` with below error:

```
[topic-fragment] Processing file:/.../tmppdf/ditamapfile/report.html
[topic-fragment] Error on line 16 column 8 of report.html:
[topic-fragment]   SXXP0003: Error reported by XML parser: XML document structures must start and end within
[topic-fragment]   the same entity.
[topic-fragment] Error 
[topic-fragment]   Error reported by XML parser processing
[topic-fragment]   file:/.../tmppdf/ditamapfile/report.html: XML document structures must start and end within the same entity.
[topic-fragment] Failed to process XML filter: Failed to transform /.../tmppdf/ditamapfile/report.html: 
Error reported by XML parser processing file:/.../tmppdf/ditamapfile/report.html: XML document structures must start and end within the same entity.
```

So, I went to that file /.../tmppdf/ditamapfile/report.html, and it is now looking as below:

```html
<?workdir /.../tmppdf/ditamapfile><?workdir-uri file:/.../tmppdf/ditamapfile/><?path2project ><?path2project-uri ./><?path2rootmap-uri ./><html xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot">
   
   
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <title>Report</title>
      <meta charset="UTF-8">
      
   </head>
   
   
   <body>
      
      <h1>This is the report</h1>
      
   </body>
   
   
</html>
```

Failing line is number 16 `</html>`.

**Comment by cinthyargc** [comment]
Looking at xml validator, the error is because meta tag is not being closed properly, missing the closing slash.

------------------------------------------------------------------------v
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"**/**>

I am unable to find where this meta tag is being added.

**Comment by cinthyargc** [comment]
Hi Radu, 

I just wanted to share, that I updated to latest version of DITA-OT, and I am now able to add html content into my HTML and PDF, I was missing the fix for #2947. 

**Important Note:**
I had to re-formart my HTML, to match HTML 5, and use article tags, otherwise, HTML body was not being added to PDF or HTML.

To anyone reaching this point, use HTML 5 tags and DITA-OT >= 3.3.2. Below a sample code:

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Report Viewer</title>
  </head>
  <body>
    <article>
      <h1>Placement Report</h1>
      <p>Report will be placed here</p>
    </article>
  </body>
</html>
```
